src/main/java/de/medizininformatik_initiative/process/report/spring/config/FhirClientConfig.java

```diff
@@ -95,6 +99,51 @@ public class FhirClientConfig
+
+	@ProcessDocumentation(processNames = {
+			"medizininformatik-initiativede_reportSend" }, description = "Secret of the client (password) used for authentication when accessing the oidc provider token endpoint", recommendation = "Use docker secret file to configure by using *${env_variable}_FILE*")
+	@Value("${de.medizininformatik.initiative.report.dic.fhir.server.oauth2.client.password:#{null}}")
+	private String fhirStoreOAuth2ClientSecret;
+
+	@ProcessDocumentation(processNames = {
+			"medizininformatik-initiativede_reportSend" }, description = "The timeout in milliseconds until a connection is established between the client and the oidc provider", recommendation = "Change default value only if timeout exceptions occur")
+	@Value("${de.medizininformatik.initiative.report.dic.fhir.server.oauth2.timeout.connect:20000}")
+	private int fhirStoreOAuth2ConnectTimeout;
+
+	@ProcessDocumentation(processNames = {
+			"medizininformatik-initiativede_reportSend" }, description = "Maximum period of inactivity in milliseconds between two consecutive data packets of the client and the oidc provider", recommendation = "Change default value only if timeout exceptions occur")
+	@Value("${de.medizininformatik.initiative.report.dic.fhir.server.oauth2.timeout.socket:60000}")
+	private int fhirStoreOAuth2SocketTimeout;
+
+	@ProcessDocumentation(processNames = {
+			"medizininformatik-initiativede_reportSend" }, description = "PEM encoded file with one or more trusted root certificate to validate the oidc provider server certificate when connecting via https", recommendation = "Use docker secret file to configure", example = "/run/secrets/hospital_ca.pem")
+	@Value("${de.medizininformatik.initiative.report.dic.fhir.server.oauth2.trust.certificates:#{null}}")
+	private String fhirStoreOAuth2TrustStore;
+
+	@ProcessDocumentation(processNames = {
+			"medizininformatik-initiativede_reportSend" }, description = "Proxy location, set if the oidc provider can only be reached through a proxy", example = "http://proxy.foo:8080")
+	@Value("${de.medizininformatik.initiative.report.dic.fhir.server.oauth2.proxy.url:#{null}}")
+	private String fhirStoreOAuth2ProxyUrl;
+
+	@ProcessDocumentation(processNames = {
+			"medizininformatik-initiativede_reportSend" }, description = "Proxy username, set if the oidc provider can only be reached through a proxy which requests authentication")
+	@Value("${de.medizininformatik.initiative.report.dic.fhir.server.oauth2.proxy.username:#{null}}")
+	private String fhirStoreOAuth2ProxyUsername;
+
+	@ProcessDocumentation(processNames = {
+			"medizininformatik-initiativede_reportSend" }, description = "Proxy password, set if the oidc provider can only be reached through a proxy which requests authentication", recommendation = "Use docker secret file to configure by using *${env_variable}_FILE*")
+	@Value("${de.medizininformatik.initiative.report.dic.fhir.server.oauth2.proxy.password:#{null}}")
+	private String fhirStoreOAuth2ProxyPassword;
+
```

Comment:
Consider using the DSF proxy config here and for other proxy variables of the process.

See for example [NUM-CODEX TransferDataConfig.java](https://github.com/num-codex/codex-processes-ap1/blob/ee36db8f658f88a29130df1a1e75a5b1cac66a80/codex-process-data-transfer/src/main/java/de/netzwerk_universitaetsmedizin/codex/processes/data_transfer/spring/config/TransferDataConfig.java#L307-L314)

Reply:
An issue for this feature already exists (#8) and will be implemented later on.

Reply:
implemented with dea7592